modules/network/smb2man/libsmb2/lib/smb2-cmd-lock.c

```
@@ -0,0 +1,312 @@
+                        smb2_set_uint64(iov, offset + 8, elements->length);
+                        smb2_set_uint32(iov, offset + 16, elements->flags);
+                        offset += SMB2_LOCK_ELEMENT_SIZE;
+                        elements++;
+                }
+        }
+
+        return 0;
+}
+
+struct smb2_pdu *
+smb2_cmd_lock_async(struct smb2_context *smb2,
```

Cppcheck (2.21.0)

[style] The function 'smb2_cmd_lock_async' is never used.

(unusedFunction)